addons/base_import/static/src/js/import.js

```diff
@@ -220,6 +220,11 @@ openerp.base_import = function (instance) {
             this.$('.oe_import_button, .oe_import_file_reload')
                     .prop('disabled', true);
             this.$el.addClass('oe_import_with_file');
+            if (!(this.$('input.oe_import_file').val().split('.').pop() == "csv")) {
+                this.$el.find('.oe_import_toggle').hide();
+            }else {
+                this.$el.find('.oe_import_toggle').show();
+            }
```

Comment:
Nope, as far as I can tell xlrd always returns unicode, and the expat parser underlying the ods reader also always returns unicode data (unless reconfigured not to by setting `returns_unicode` to `False`)